package/MDAnalysis/lib/distances.py

```diff
@@ -450,16 +450,23 @@ def capped_distance(reference: Union[np.ndarray, 'AtomGroup'],
             raise ValueError("Box Argument is of incompatible type. The "
                              "dimension should be either None or of the form "
                              "[lx, ly, lz, alpha, beta, gamma]")
-    method = _determine_method(reference, configuration, max_cutoff,
-                               min_cutoff=min_cutoff, box=box, method=method)
-    return method(reference, configuration, max_cutoff, min_cutoff=min_cutoff,
-                  box=box, return_distances=return_distances)
+
+    # The check_coords decorator made sure that reference and configuration
+    # are arrays of positions. Mypy does not know about that so we have to
+    # tell it.
```

Comment:
It would be better to address that directly in the decorator, but I am not sure how to do that or if it is even possible.